oci/archive/oci_src.go

```diff
@@ -43,7 +43,7 @@ func LoadManifestDescriptor(imgRef types.ImageReference) (imgspecv1.Descriptor,
 	if !ok {
 		return imgspecv1.Descriptor{}, errors.Errorf("error typecasting, need type ociArchiveReference")
 	}
-	tempDirRef, err := createUntarTempDir(ociArchRef)
+	tempDirRef, err := createUntarTempDir(nil, ociArchRef)
```

Comment:
This should pass a real context instead of `nil`, https://github.com/containers/image/pull/747#discussion_r346954903 still outstanding.

Reply:
The problem is that `LoadManifestDescriptor` is not used, so i see not way how to pass the systemContext here

Reply:
`LoadManifestDescriptor` exists to be called from other software, notably https://github.com/containers/libpod/blob/c673ff8cb63c71a39e1a1d561373af42bfea7f24/libpod/image/pull.go#L178 (which is what #495 wants to improve).

The technique should be the same as for other public functions - introduce new `LoadManifestDescriptorWithContext`, provide a compatibility wrapper `LoadManifestDescriptor` that calls the new one (and eventually update Podman to call the new one and benefit from the feature).